widget-demo.html

```diff
@@ -51,6 +51,15 @@
 			j.src = 'build/rocketchat-livechat.min.js?_=' + Math.random();
 			h.parentNode.insertBefore(j, h);
 		})(window, document, 'script', 'http://localhost:8080');
+		RocketChat(function() {
```

Comment:
I know this isn't related to this PR, but still, I've purposefully added to show a demo about how to use the LiveChat widget API's